### DISCUSSION TITLE
How is working actor supervision?

### DISCUSSION THREAD

**Discussion by EnvOut** [slawlor/ractor#82] [Q&A] [answered]
Readme mentioned actor supervision, but I can't find any samples / description in docs 
Could you please provide how it is works?

**Comment by slawlor** [comment, marked as answer]
You can start with [supervisor.rs](https://github.com/slawlor/ractor/blob/main/ractor/examples/supervisor.rs) for some basic ergonomics. Every actor can be a supervisor to other actors by linking them together, similar to the Erlang naming conventions. You can additionally `spawn_linked` which will spawn and immediately link to the child actor to the parent supervisor automatically.
